Automattic/jetpack
# Use the Password_Checker package

Fixes #
Depends on #18700.

#### Changes proposed in this Pull Request:

* Uses the Password_Checker package.
* Removes the usage of jetpack_require_lib().

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Open wp shell for a Jetpack site. 
* `yarn docker:wp shell --url=DOMAIN`
* wp_set_current_user( 1 );
* $um = Automattic\Jetpack\Sync\Modules::get_module( 'users' );
* $um->authenticate_handler( wp_get_current_user(), wp_get_current_user()->user_login, 'test' );
* Verify WP_User is returned.

#### Proposed changelog entry for your changes:


* Sync: Use the new Password_Checker package instead of Jetpack_Password_Checker.
